index.html

```diff
@@ -81,25 +82,101 @@ <h1 class="main-head">
               />
             </div>
           </div>
-          <div class="d-flex align-items-center rectangle-with-icons">
-            <ul>
-              <li>
-                <div class="circle"></div>
-                <span>90+</span>
-                <p>Users</p>
+          <div class="box-flex">
+            <div class="box-shadow" aria-hidden="true"></div>
+            <ul class="white-box">
+              <li class="icons-box-item">
+                <div class="flex-item">
+                  <span
+                    class="icon-user_icon position-icons"
+                    aria-hidden="true"
+                  ></span>
+                </div>
+                <div>
+                  <span class="big-numbers">90+</span>
+                  <p class="coments-for-icons">Users</p>
```

Comment:
Why do you use `p` for this tiny text?

Reply:
It's first element that I thought about))
I have to use span, right?

Reply:
If i don't use p element, I have problems with my position. Numbers and word User positioned in one line.

Reply:
You can use `span` and `display: inline-block`.